src/lib/docs/ContractKit/ActionMethod.md

```diff
@@ -1,9 +1,47 @@
 ---
 title: Action (Method)
-description: change_me
+description: Provides functionality for genereting an Antelope Action instance that can then be passed to a a session.transact() method.
 category: ContractKit
 published: true
 order: 3
 ---
 
-Using the contract.action call
+# Action
+
+The `action` method assists in generating an Antelope [Action](/docs/antelope/action) instance. Actions can be created using this method then be passed to the [SessionKit transact](/docs/session-kit/transact) method to be executed on chain.
+
+## Usage
+
+With an instance of the `Contract` class, you can use the `action` method to generate an action instance.
+
+```typescript
+const action = tokenContract.action(
+    'transfer', // or Name.from('transfer')
+    {
+        from: 'foo',
+        to: 'bar',
+        quantity: '1.0000 EOS',
+    },
+    {
+        authorization: [
```

Comment:
When writing the first example of something like this, I have been trying to make it the most simple example - and excluding all of the optional parameters. So for this I'd take out the 3rd parameter from the action call to keep it simple, and then add a new section further down the document that talks about the options.

I left this comment here, but I'd take a look at all the documents and try to apply this same logic to them.